pyquil/device.py

```diff
@@ -270,31 +270,40 @@ def __neq__(self, other):
 class Device(object):
     """
     A device (quantum chip) that can accept programs. Only devices that are online will actively be
-    accepting new programs.
+    accepting new programs. In addition to the self.raw attribute, two other keys are optionally
+    expected in device._raw - 'isa' and 'noise_model' - which should conform to the dictionary
```

Comment:
> In addition to the self.raw attribute, two other keys are optionally expected in device._raw - 'isa' and 'noise_model'

I'm having trouble parsing this. What is a self.raw attribute? How does this compare to the keys? Why is it underscore-prefixed once and not the other

Reply:
@mpharrigan Thanks, I've cleaned up the description a bit. There is just `self._raw` (no `self.raw`), and the other attributes, `.isa` and `.noise_model` are constructed from keys in `._raw`.

Reply:
@ntezak I've resolved the issue I was alluding to last night regarding the `KrausModel` test. @pkaralekas this is ready for a second pass.